### ISSUE TITLE
ms_simplify giving Error in ogrInfo : Cannot open data source

### ISSUE THREAD

**Issue by boydcurrey** [ateucher/rmapshaper#74]
I have been using `ms_simplify` to simplify shapefiles for the past few weeks, and then I went away for a week on holiday. I come back to pickup my code, and I now get an error when calling `ms_simplify`:

```
Error in ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv,  : 
  Cannot open data source
```

This is how I'm using it as per various guides out there:

```
Map.Country.Source <- readOGR(dsn = "./Data/Mapping/1270055001_aus_2016_aust_shape", layer = "AUS_2016_AUST", dropNULLGeometries = T, stringsAsFactors = F)
Map.Country.Source@data$id <- rownames(Map.Country.Source@data)

Map.Country.Source <- ms_simplify(Map.Country.Source, keep = 0.1, keep_shapes = T)
```

In case it matters (though it's happening to me on various shapefiles), the shapefile referred to above is  available at:
http://www.abs.gov.au/AUSSTATS/subscriber.nsf/log?openagent&1270055001_aus_2016_aust_shape.zip&1270.0.55.001&Data%20Cubes&5503B37F8055BFFECA2581640014462C&0&July%202016&24.07.2017&Latest

Any ideas on why this is happening? I found the following article while looking for a solution and it was published on the day I went on holiday. Is this related?

https://cran.r-project.org/web/packages/rgdal/vignettes/OGR_shape_encoding.pdf

Thanks for your help :)

**Comment by boydcurrey** [comment]
Some additional info from the stack trace:

Error in ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv, : Cannot open data source

```
12. ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv, swapAxisOrder = swapAxisOrder, require_geomType = require_geomType)
11. withCallingHandlers(expr, message = function(c) invokeRestart("muffleMessage"))
10. suppressMessages(ogr_info <- ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv, swapAxisOrder = swapAxisOrder, require_geomType = require_geomType))
9. rgdal::readOGR(geojson, "OGRGeoJSON", verbose = FALSE, disambiguateFIDs = TRUE, p4s = proj, stringsAsFactors = FALSE, encoding = enc, use_iconv = enc != "unknown")
8. withCallingHandlers(expr, message = function(c) invokeRestart("muffleMessage"))
7. suppressMessages(rgdal::readOGR(geojson, "OGRGeoJSON", verbose = FALSE, disambiguateFIDs = TRUE, p4s = proj, stringsAsFactors = FALSE, encoding = enc, use_iconv = enc != "unknown"))
6. withCallingHandlers(expr, warning = function(w) invokeRestart("muffleWarning"))
5. suppressWarnings(suppressMessages(rgdal::readOGR(geojson, "OGRGeoJSON", verbose = FALSE, disambiguateFIDs = TRUE, p4s = proj, stringsAsFactors = FALSE, encoding = enc, use_iconv = enc != "unknown")))
4. GeoJSON_to_sp(ret, proj = attr(geojson, "proj4"))
3. ms_sp(input, call)
2. ms_simplify.SpatialPolygons(Map.Country.Source, keep = 0.1, keep_shapes = T)
1. ms_simplify(Map.Country.Source, keep = 0.1, keep_shapes = T)

```

**Comment by ateucher** [comment]
Hi @boydcurrey. This is due to a change in the latest version of rgdal (you can see a discussion [here](https://github.com/ropensci/geojsonio/issues/130) as it also affected geojsonio). I have an update of rmapshaper ready to submit to CRAN, it will hopefully be published in the next few days. In the meantime, you should be able to use the development version of `rmapshaper` with the development version of `geojsonio`:

```r
# install.packages("devtools")`
devtools::install_github("ropensci/geojsonio")
devtools::install_github("ateucher/rmapshaper")
```

**Comment by boydcurrey** [comment]
Thanks @ateucher for the quick response. I will have a read and give these a whirl and look forward to the updated release. Thanks again :)

**Comment by ateucher** [comment]
@boydcurrey this is now fixed in rmapshaper v0.4, now on CRAN
